spec/models/customer_spec.rb

```diff
@@ -4,58 +4,13 @@
   describe "relationships" do
     it { should have_many :invoices}
     it { should have_many(:transactions).through(:invoices) }
-    # it { should have_many(:merchants).through(:invoices) }
   end
 
   describe "validations" do
     it { should validate_presence_of :first_name }
     it { should validate_presence_of :last_name }
   end
 
-  describe "#total_transactions" do
```

Comment:
Since we still have a method defined for total transactions (now buried in another method) we still need this to test it directly

Reply:
confirmed with Juliet that we are good like this